source/includes/aggregation/aggExpressions.kt

```diff
@@ -0,0 +1,529 @@
+import com.mongodb.client.model.*
+import com.mongodb.client.model.mql.*
+import com.mongodb.client.model.mql.MqlValues.current
+import com.mongodb.client.model.mql.MqlValues.of
+import com.mongodb.kotlin.client.MongoClient
+import org.bson.Document
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+import java.time.LocalDate
+
+class AggExpressionsTest {
+    companion object {
+        val uri = "<connection string>"
+        val mongoClient = MongoClient.create(uri)
+        private val database = mongoClient.getDatabase("aggExpressions")
+
+        @AfterAll
+        @JvmStatic
+        fun afterAll() {
+            mongoClient.close()
+        }
+    }
+
+    @Test
+    fun arithmeticOpsTest() {
+
+        data class Entry(val date: LocalDate, val precipitation: Double)
+
+        val collection = database.getCollection<Entry>("weatherData")
+
+        val entries = listOf(
+            Entry(LocalDate.of(2021, 6, 24), 5.0),
+            Entry(LocalDate.of(2021, 6, 25), 1.4)
+        )
+
+        collection.insertMany(entries)
+
+        // start-arithmetic-aggregation
+        val month = current().getDate("date").month(of("UTC"))
+        val precip = current().getInteger("precipitation")
+
+        val results = collection.aggregate<Document>( // :remove:
```

Comment:
Are these left in on purpose?

Reply:
Oops! Leftover from the kotlin coroutine bluehawk code